code/modules/surgery/bodyparts/bodypart_effects.dm

```diff
@@ -0,0 +1,138 @@
+/// For scaling the effectiveness of certain effects to the total bodypart count
+#define GET_BODYPART_COEFFICIENT(X) round(X.len / 6, 0.1)
+/// Check if it's full body. These are mostly here so we can change just one place when we ever add more limbs (?)
+#define IS_FULL_BODY(X) (X.len == 6)
```

Comment:
This should either use
A. `species.max_bodypart_count` (technically, species can have varying numbers of bodyparts)
or
B. `BODYPARTS_DEFAULT_MAXIMUM` (which is just 6)